.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "deps/cziplib"]
 	path = deps/cziplib
 	url = https://github.com/kuba--/zip.git
+[submodule "deps/PowerAccent"]
```

Comment:
I don't think this will be acceptable. We can't maintain 2 projects this way, if something breaks in PowerAccent side, we can't fix it. Also if we e.g. want to move it to WinUI at some point it's not on our side to do it 

@crutkas ?

Reply:
I see PowerAccent as a complete project, not at a library, with evolution in the future.
I can implement with a "copy/paste" the complete code, but that mean it will be not so easy to make it changeable as the main repo.

btw, is it possible to fix the submodule to a specific version? And avoid any break. I can create a branch for it if it can help.

Reply:
As a team, i think we'd want to fork / copy.  If we'd needed to upgrade something (example when we did .NET 6 upgrades and ARM64 support), you may not want to.  Same with accessibility things, we have to do trade offs to be accessible.

Translation will be another thing I can see.

Reply:
OK, thanks for the answers. I will do that!